mozzarilla/windows/tag_converters/model_converter.py

```diff
@@ -148,12 +149,15 @@ def destroy(self):
         window_base_class.destroy(self)
 
     def convert(self, tag_path):
+        # for compatibility with stubbs, we'll read as stubbs mode(since
+        # stubbs simply adds fields), but we'll write as halo mode(no one is
+        # creating stubbs models, so they'll want everything as halo models)
         if self.to_gbxmodel:
-            src_tag = mode_def.build(filepath=tag_path)
+            src_tag = stubbs_mode_def.build(filepath=tag_path)
             dst_tag = mod2_def.build()
         else:
             src_tag = mod2_def.build(filepath=tag_path)
-            dst_tag = mode_def.build()
+            dst_tag = halo_mode_def.build()
```

Comment:
I'm not into the implicit assumption that the output format should be a Halo `mode` because nobody makes stubbs models, but the MEK doesn't advertise itself as a Stubbs editing kit so whatever.

Reply:
Yeah. I do want to eventually be able to do more for stubbs modding. But that's a ways away and by that time I'm going to assume that everything only works with Halo anyway.